redbadger/count-dracula
# Nhs UI library

Closes #4 

## Technical acceptance criteria

- [x] Add via NPM official NHS UI library

### How to test

- Run `yarn workspace web start`
- Go to [localhost:3000](localhost:3000)
- Check it render an NHS component

![](https://media.giphy.com/media/VGh9QVwgUPj3aL063N/giphy.gif)
